app/decorators/products_decorator.rb

```diff
@@ -3,8 +3,18 @@ class ProductsDecorator < Draper::CollectionDecorator
 
   def to_csv
     CSV.generate do |csv|
-      csv << Product.attribute_names
-      each { |product| csv << product.to_csv }
+      csv << attributes_for_export
+      each { |product| csv << product_to_csv(product) }
     end
   end
+
+private
+
+  def product_to_csv(product)
+    attributes_for_export.map { |key| product.send(key) }
+  end
+
+  def attributes_for_export
+    Product.attribute_names.dup.concat(%w[case_ids])
```

Comment:
`attribute_names` is sorted alphabetically so we can ensure this isn't shown out of order.

```suggestion
    Product.attribute_names.dup.concat(%w[case_ids]).sort
```